freqtrade/misc.py

```diff
@@ -211,6 +218,13 @@ def build_subcommands(parser: argparse.ArgumentParser) -> None:
         type=int,
         metavar='INT',
     )
+    hyperopt_cmd.add_argument(
+        '-tp', '--timeperiod',
```

Comment:
Please use one character names for single hyphen parameters.

> In the original Unix tradition, command-line options are single letters preceded by a single hyphen.

See http://www.faqs.org/docs/artu/ch10s05.html